presto-iceberg/src/test/java/com/facebook/presto/iceberg/IcebergDistributedSmokeTestBase.java

```diff
@@ -891,6 +891,23 @@ public void testPartitionedByDoubleWithNaNInf()
         dropTable(session, tableName);
     }
 
+    @Test
+    public void testFilterBySubfieldOfRowType()
```

Comment:
Do you think this test case would fit better in `TestIcebergLogicalPlanner`?

Reply:
I notice test cases in ``TestIcebergLogicalPlanner `` are all about the verification of optimized plans, but my test case ``testFilterBySubfieldOfRowType()`` is about the verification of query result. I think it's not so suitable to be placed in ``TestIcebergLogicalPlanner ``.

Or do you means we should also verify the optimized plan of the statement?

Reply:
I think we should also validate the plan in `TestIcebergLogicalPlanner`. 

Maybe the below cases:
* Filter contains only identity partition columns
* Filter contains a partition column and a regular column
* Filter contains a partition column and a subfield of a column
* Filter contains only a subfield of a column

Reply:
This test(`testFilterBySubfieldOfRowType`) can remain in this class

Reply:
Done!